wagtail/core/blocks/field_block.py

```diff
@@ -721,6 +721,18 @@ def widget(self):
         return AdminPageChooser(target_models=self.target_models,
                                 can_choose_root=self.can_choose_root)
 
+    def get_form_state(self, value):
+        value_data = self.widget.get_value_data(value)
+        if value_data is None:
+            return None
+        else:
+            return {
+                'id': value_data['id'],
```

Comment:
For future: We probably should make this representation to match the Admin API so the same code could work with both